packages/typespec-ts/src/index.ts

```diff
@@ -248,8 +249,8 @@ export async function $onEmit(context: EmitContext) {
 
     const isMultiClients = modularCodeModel.clients.length > 1;
 
+    emitTypes(dpgContext.sdkPackage, { sourceRoot: modularSourcesRoot });
```

Comment:
feel like this change is moving the models outside of the subclients folder already? https://github.com/Azure/autorest.typescript/pull/2556 should I still wait for this pr to be merged ?